### DISCUSSION TITLE
--watchdog exit

### DISCUSSION THREAD

**Discussion by serdarevica** [Lolliedieb/lolMiner-releases#1359] [Q&A]
Hi Guys,

I have miner exited but never restarted.
--watchdog exit is in bat file on the end of the code.

Is placement of --watchdog exit important?
Why do you think miner never restarted?

Thanks

**Comment by serdarevica** [comment]
Can somebody please help?

**Comment by bitcoincff** [comment]
Hi, the watchdog not work
"**Closing miner and trying to call external script:  reboot.sh (--watchdog script)**"
lolminer 1.46a
Run as root
Simplemining
